lib/heap_sort.rb

```diff
@@ -1,8 +1,20 @@
+require_relative "./min_heap"
 
 
 # This method uses a heap to sort an array.
-# Time Complexity:  ?
-# Space Complexity: ?
-def heap_sort(list)
-  raise NotImplementedError, "Method not implemented yet..."
+# Time Complexity:  n(log(n))
+# Space Complexity: O(1)
+def heapsort(list)
```

Comment:
How do you have a space complexity of O(1)?  You're building a new heap of n elements...

Otherwise this works.